packages/web3-providers-ws/package.json

```diff
@@ -1,14 +1,13 @@
 {
     "name": "web3-providers-ws",
-    "namespace": "ethereum",
     "version": "1.0.0-beta.37",
     "description": "Module to handle web3 RPC connections over WebSockets.",
-    "repository": "https://github.com/ethereum/web3.js/tree/master/packages/web3-providers-ws",
+    "repository": "https://github.com/ethereum/web3.js/tree/1.x/packages/web3-providers-ws",
     "license": "LGPL-3.0",
     "main": "src/index.js",
     "dependencies": {
-        "underscore": "1.8.3",
+        "underscore": "1.9.1",
         "web3-core-helpers": "1.0.0-beta.37",
-        "websocket": "git://github.com/frozeman/WebSocket-Node.git#browserifyCompatible"
+        "websocket": "github:frozeman/WebSocket-Node#browserifyCompatible"
```

Comment:
It looks like you've switched this back to the version that was incorrectly packed. That version includes a `.git` directory in the published module, which results in npm errors in certain cases (e.g. when running `npm install` a second time).

A few people in this PR already expressed relief at this error going away; it was the part I was most looking forward to about this release personally. Could you elaborate on why this update was reverted? I'd be happy to help out if you ran into an issue you don't have time to investigate.

Reply:
I've changed it back to the GitHub repository of @frozeman because I couldn't figure out why this changed was required then. Was there a clear reason for it? @frozeman

Reply:
I found some additional context about that: https://github.com/theturtle32/WebSocket-Node/pull/314
I wonder if the issue this was meant to fix (https://github.com/theturtle32/WebSocket-Node/issues/304) is still an issue or not - that library has changed a great deal since that report. So has Edge, for that matter.

Reply:
I've added the ``preinstall`` command to the ``package.json`` file of the ``web3-providers-ws`` module which will fix it. I will get in contact with the former maintainer to get the missing information.

Reply:
Great, thanks! That avoids the error I was seeing before at least.

I did test this on Edge (using browserify, as the PR described) just to see if it's still an issue, but I wasn't able to reproduce the problem. So I'm guessing whatever behavior that was meant to workaround in Edge 11 is no longer present in Edge 18.

Reply:
The ``websocket`` package will get ignored for the minified file which will be used in the browser it just has an impact if it's running on node. Fabian just sent me the following explanation for it: 
``` 
There was a problem with the meteor build cloentnor snother packager. So I had to fork an makebthis change
It is important 
```

Reply:
The change in question was made solely to the `browser.js` file, so I don't believe it's a node issue per se - it was a problem with the browserify step itself, which of course is run using node. You can see [the commit](https://github.com/frozeman/WebSocket-Node/commit/7004c39c42ac98875ab61126e5b4a925430f592c) has the message `made browserify compat`.

So I believe the way I tried to reproduce it was correct. Though from the sounds of things, maybe it was something particular to that build process that resulted in the error, rather than just the browser used.